src/app/screens/Search/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import SearchEmptyFallback from '@components/SearchEmptyFallback';
+import { AppState } from '@interfaces/appState';
+import { Book } from '@interfaces/book';
+import BookList from '@screens/BookList';
```

Comment:
You shouldn't import this screen from here. I think the best way to do this is to create the BookList as a base component
Then you have 2 screens:
- one for BookList that gets the books from backend and renders the `BookList` component
- one for Search, which filters the books that get passed to the `BookList` component and passes the `EmptyComponent` too.